### DISCUSSION TITLE
https://theme-hope.vuejs.press/zh/guide/interface/others.html

### DISCUSSION THREAD

**Discussion by giscus** [vuepress-theme-hope/giscus-discussions#927] [Announcements]
# https://theme-hope.vuejs.press/zh/guide/interface/others.html

打印按钮 试一试: 主题对打印样式进行了全面优化，默认在桌面模式的目录下会有一个打印按钮。 要隐藏打印按钮，你应该在主题选项中设置 print: false。 全屏按钮 试一试:

https://theme-hope.vuejs.press/zh/guide/interface/others.html

**Comment by Alex6357** [comment]
laptop布局下能不能也显示打印按钮？

**Comment by mtueih** [comment]
太专业了👍

**Comment by YatsukiRenka** [comment]
你好, 我想问一下, 打印为PDF时表格和mermaid显示不完整有什么解决方法吗?

**Reply by Mister-Hope** [reply]
你可以开一个新的议题，附上会出问题的markdown以及网页。
